data/50-default.json

```diff
@@ -85,5 +85,26 @@
     "ref-kind": "runtime",
     "name": "com.endlessm.CompanionAppService.Locale",
     "branch": "stable"
+  },
+  {
+    "action": "install",
+    "serial": 2019102100,
+    "ref-kind": "app",
+    "collection-id": "com.endlessm.Apps",
+    "remote": "eos-apps",
+    "name": "com.hack_computer.Clubhouse",
+    "branch": "eos3",
+    "filters": {
+        "architecture": ["x86_64"]
+    }
+  },
+  {
+    "action": "uninstall",
+    "serial": 2019102200,
+    "ref-kind": "app",
+    "collection-id": "com.endlessm.Apps.Hack",
```

Comment:
uninstall actions don't need collection IDs or remotes, they match the local app ID

Reply:
Good catch, resolved.